src/Lives/Lives.test.js

```diff
@@ -11,10 +11,10 @@ import { createStore } from 'redux';
 
 const store = createStore(() => ({setLives: 1}))
 
-describe('Header', () => {
+describe('Lives', () => {
 
 	it('should render without crashing', () => {
-		const { getByText } = render(
+		const { getByText, getAllByTestId } = render(
 				<BrowserRouter>
 					<Provider store={store}>
             <Header 
```

Comment:
Why is the Header still being rendered here? For a unit test like this, you want to make sure that the component under test can be successfully rendered in isolation. It looks like you could just render `<Lives lives={2}/>` without the `Header`, and still make the same assertions ( maybe without lines 30 and 33).